lib/tasks/foreman_templates_tasks.rake

```diff
@@ -57,6 +57,13 @@ namespace :templates do
     }).purge!
   end
 
+  desc 'Clean data created by this plugin, this will permanently delete the data!'
```

Comment:
Fair enough. I see this being run automatically in the future anyway, so lets not get dragged down over wording :)